pkg/controller/main.go

```diff
@@ -226,7 +227,7 @@ func Main(f *Flags, version string) error {
 			}
 			if ns != namespace {
 				ssinf = ssinformers.NewFilteredSharedInformerFactory(ssclientset, 0, ns, tweakopts)
-				sinf = informers.NewFilteredSharedInformerFactory(clientset, 0, ns, tweakopts)
+				sinf = initSecretInformerFactory(clientset, ns, tweakopts, f.SkipRecreate)
```

Comment:
This is for additional namespaces, but we are also creating a secrets informer factory for the "default" namespace. Can you check that one too?

Reply:
checking

Reply:
fixed